specification.md

```diff
@@ -4,73 +4,73 @@ Opportunity:
 
 * The programming language and toolchain used to analyse single cell data determines the format that the data will be published in. These language- and toolchain-driven data silos inhibit use cases like model training that require bringing data to tools.
 * Best-in-class algorithms are often available in only a single language ecosystem or toolchain, or take substantial effort to make portable.
-* Multimodal data, which are becoming more common, lack standardization and support--particularly in the python ecosystem.
-* Data are becoming large enough that moving serialized objects around will soon be infeasible - cloud optimized formats will be required to support the next analysis phase, and out of core processing is becoming increasingly important.
+* Multimodal data, which are becoming more common, lack standardization and support -- particularly in the Python ecosystem.
+* Data are becoming large enough that moving serialized objects around will soon be infeasible -- cloud-optimized formats will be required to support the next analysis phase, and out-of-core processing is becoming increasingly important.
 
-We envision an API that enables users to slice and compute on large (100s millions observations) single cell datasets stored in the cloud using the AnnData, Seurat, or Bioconductor toolchains.
+We envision an API that enables users to slice and compute on large (100s of millions of observations) single-cell datasets stored in the cloud using the AnnData, Seurat, or Bioconductor toolchains.
 
-To get started, we will focus on enabling data to be used by all major toolchains and on nailing the multi-modal data use cases. We believe this will be sufficient to drive API adoption. With data silos broken, larger data use cases will be enabled and the need for a cloud-optimized data format will be more widely felt.
+To get started, we will focus on enabling data to be used by all major toolchains and on nailing the multimodal-data use cases. We believe this will be sufficient to drive API adoption. With data silos broken, larger-data use cases will be enabled and the need for a cloud-optimized data format will be more widely felt.
```

Comment:
Initial focus and longer-term goals do not belong in a specification. They could be included in the README for the site if we wanted to advertise a roadmap, but I don't believe that's required. I'd move from the Introduction (why we're doing this) to the Data Model.